src/lib/widget/inline.svelte

```diff
@@ -16,75 +16,25 @@
   export const journey = api.journey;
   export const request = api.request;
   export const user = api.user;
+
+  export type ConfigurationApi = ReturnType<typeof api.configuration>;
```

Comment:
❤️

Reply:
Thanks to your teaching, I was able to think of this solution.